cpp/tests/utilities/cudf_test_utils.cuh

```diff
@@ -56,11 +116,11 @@ void print_valid_data(const gdf_valid_type *validity_mask,
 
 /* --------------------------------------------------------------------------*/
 /**
- * @brief  Creates a unique_ptr that wraps a gdf_column structure intialized with a host vector
+ * @Synopsis  Creates a unique_ptr that wraps a gdf_column structure intialized with a host vector
```

Comment:
Please use `@brief` as was used before (and everywhere else).

Reply:
This was a mistake, I didn't mean for that change to occur.